std/uni.d

```diff
@@ -1898,23 +1898,29 @@ public alias CodepointSet = InversionList!GcPolicy;
 */
 public struct CodepointInterval
 {
-pure:
-    uint[2] _tuple;
-    alias _tuple this;
+    private uint[2] _tuple;
+    alias tuple this;
 
 @safe pure nothrow @nogc:
-
+    /// Constructor
     this(uint low, uint high)
     {
         _tuple[0] = low;
         _tuple[1] = high;
     }
+
+    /// Support for equality testing
     bool opEquals(T)(T val) const
     {
         return this[0] == val[0] && this[1] == val[1];
     }
+
+    /// Access to the interval members
     @property ref inout(uint) a() inout { return _tuple[0]; }
+    /// ditto
     @property ref inout(uint) b() inout { return _tuple[1]; }
+    /// ditto
+    @property ref inout(uint[2]) tuple() inout { return _tuple; }
```

Comment:
I don't know how much sense it makes to add this, but it's downright weird to be returning a static array and call it `tuple`.

I assume that the reason that you're adding this is because you can't alias `_tuple` to `this` and have `_tuple` be private while the alias is public? Why not just rename `_tuple` to `tuple` and leave it as public then? The alias seems to have been intended to allow mutation, which this change breaks. Regardless, if we're going to put what's being aliased in the docs, then we should probably pick a better name than `tuple` given that it's a static array.

I confess though that I'd be more inclined to just make it clear in the documentation that `CodePointInterval` can be used like a tuple and not put any any of these members in the documentation. They're not actually needed to make the usage clear, since it's pretty clearly supposed to be a named `Tuple`. In fact, I find it a bit odd that `a` and `b` even exist, since they're redundant with `[0]` and `[1]`, though I guess that that's in line with doing something like `Tuple!(uint, "a", uint, "b")`.